src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js

```diff
@@ -90,11 +155,11 @@ class SmartContent extends React.Component<Props> {
         this.filterCriteriaChangeDisposer();
     }
 
-    handleFilterCriteriaChange = () => {
+    handleFilterCriteriaChange = (filterCriteria: ?FilterCriteria) => {
         const {onChange, onFinish, value} = this.props;
 
         const currentValue = {...filterCriteriaDefaults, ...toJS(value)};
-        const newValue = {...filterCriteriaDefaults, ...toJS(this.smartContentStore.filterCriteria)};
+        const newValue = {...filterCriteriaDefaults, ...toJS(filterCriteria)};
```

Comment:
is the `filterCriteriaDefaults` constanct still needed?

Reply:
Had it already delete, but then I had to revert a change :sweat_smile: Gonna fix it.